content/lessons/firebase-emulator-advanced/index.md

```diff
@@ -82,16 +82,16 @@ if (process.env.NODE_ENV !== 'production') {
 
 ### What cannot be Emulated?
 
-Auth, Storage, and FCM are not currently supported. If you need to create a large number of mock records for these services, consider creating a second "sandbox" Firebase project. 
+Auth, Storage, and FCM are not currently supported. If you need to create a large number of mock records for these services, consider creating a second "sandbox" Firebase project.
 
 
 ## Mock Data
 
-The emulator makes the Firebase Admin SDK available on the browser `window`. This means you can just call `firestore.doc(...).set(...)` from a client-side script or the browser console. 
+The emulator makes the Firebase Admin SDK available on the browser `window`. This means you can just call `firestore.doc(...).set(...)` from a client-side script or the browser console.
 
 ### Generate Fake Firestore Data Quickly
 
-We can quickly generate mock data by injecting [Faker.js](https://github.com/marak/Faker.js/) into a browser script. The script appends a button to the UI, then tells Firestore to create 100 user documents with fake data. Run it by simply pasting this code into the browser console. 
+We can quickly generate mock data by injecting [Faker.js](https://github.com/faker-js/faker) into a browser script. The script appends a button to the UI, then tells Firestore to create 100 user documents with fake data. Run it by simply pasting this code into the browser console.
```

Comment:
I edited the file with cloud VS Code, provided by the Github platform, which automatically improved the Prettier formatting.
This line includes the actual url change!